.github/workflows/ci.yml

```diff
@@ -0,0 +1,105 @@
+name: CI
+env:
+  node-version: 12
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Install Dependencies (Linux)
+      if: matrix.os == 'ubuntu-latest'
+      run: sudo apt-get install fakeroot dpkg rpm lintian gnupg2 reprepro createrepo xvfb openssl
+    - name: Install Dependencies (macOS)
+      if: matrix.os == 'macos-latest'
+      run: brew install graphicsmagick imagemagick
```

Comment:
hy we need `imagemagick` for macos?

Reply:
It's used by `create-dmg` for the icon apparently. I guess that if we don't use it, we'll just get the default DMG icon rather than one with Headset logo.
https://github.com/sindresorhus/create-dmg#dmg-icon